xunit-smoketest/test.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
+set -euo pipefail
+
 for TEST_RESULT in pass fail ;
 do
   export TEST_RESULT
-  OUTPUT=$(dotnet test 2>&1)
-  EXIT_CODE=$?
+  EXIT_CODE=0
+  OUTPUT=$(dotnet test 2>&1) || EXIT_CODE=$?
```

Comment:
Maybe a `;` should be better instead of `||`? We want to always set the exit code, don't we?

Reply:
The ` || ` is to avoid exiting immediately when `dotnet test` fails (due to `set -e`).